README.md

```diff
@@ -1,6 +1,6 @@
 # docker-wkhtmltopdf-aas
 [![License (3-Clause BSD)](https://img.shields.io/badge/license-BSD%203--Clause-brightgreen.svg)](http://opensource.org/licenses/BSD-3-Clause)
-[![Build Status](https://travis-ci.org/traum-ferienwohnungen/docker-wkhtmltopdf-aas.svg?branch=master)](https://travis-ci.org/traum-ferienwohnungen/docker-wkhtmltopdf-aas)
+[![Build Status](https://travis-ci.org/leknoppix/docker-wkhtmltopdf-aas.svg?branch=master)](https://travis-ci.org/leknoppix/docker-wkhtmltopdf-aas)
```

Comment:
I think the travis CI pipeline should match with the repository. I will add travis for this repo if it does not exist yet.